lib/capistrano/tasks/rom_migrate.rake

```diff
@@ -9,6 +9,7 @@ namespace :deploy do
     on roles(:db) do
       within release_path do
         with rails_env: fetch(:rails_env) do
+          execute :bundle, :exec, :rake, 'rom:migrations_applied'
```

Comment:
We want to run this rake task for a 1st time setup. I ran it on staging when I deployed with capistrano this PR. 
We should keep it for when we run it in production for the first time. 
After that it is safe to remove it.